src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$transition-smooth: 300ms;
+$turquoise: #00acdc;
+$black: #060b35;
+$gray-border: #f3f3f3;
+$gray: #616070;
+$white: #fff;
+$blue-black: #34568b;
+$font-size-standart: 12px;
```

Comment:
There is a typo in the variable name `$font-size-standart`. It should be `$font-size-standard` to correctly reflect the intended meaning.